src/core/server/execution_context/execution_context_container.test.ts

```diff
@@ -37,6 +37,17 @@ describe('KibanaExecutionContext', () => {
       expect(value).toMatchInlineSnapshot(`"1234-5678"`);
     });
 
+    it('returns an escaped string representation of provided execution contextStringified', () => {
+      const context: KibanaServerExecutionContext = {
+        id: 'Visualization☺漢字',
+        type: 'test-type',
+        requestId: '1234-5678',
+      };
+
+      const value = new ExecutionContextContainer(context).toString();
+      expect(value).toMatchInlineSnapshot(`"1234-5678"`);
```

Comment:
I think we need to add the `name` property to the `context` so we get the encoded string here? It's only including the `requestId` right now

Reply:
Fixed in 166a4e1996c